perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -352,6 +445,38 @@ def ReadObjects(storage_schema, bucket, objects_to_read, latency_results=None,
       logging.exception('Failed to read object %s', object_name)
 
 
+def ReadObject(storage_schema, bucket, object, host_to_connect=None):
```

Comment:
Done. I also renamed "bucket" to "bucket_name", because it matches and is more descriptive.